hack/preview-template.env

```diff
@@ -13,16 +21,9 @@ export MY_GITHUB_TOKEN=
 # Optional
 
 ## Cluster and KCP config
-### Set cluster kubeconfig so it's not needed to pass it as parameter
-### example: $HOME/.kube/config
-export CLUSTER_KUBECONFIG=
-### KCP cluster kubeconfig so it's not needed to pass it as parameter
-### This file is managed by preview script
-### example: $HOME/.kube/preview-kcp
-export KCP_KUBECONFIG=
-### Path to CPS kubeconfig which will be copied when using preview-cps mode
-### example: $HOME/.kube/cps
-export CPS_KUBECONFIG=
+### KCP Workspace to deploy, default "~"
```

Comment:
you wrote that the default value is `~`  which is not true, because you don't set it for the preview mode anywhere. 
Actually, the default value is `root` which is set in `flags.sh` script

Reply:
updated, default is root, mentioned that it's possible to use `~` for home workspace